change/@ni-nimble-components-af9b983d-43db-46f7-a204-c271899c2559.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What's the expected situation for keyboard interactions to expand/collapse rows in this PR? If it's not planned to be implemented yet, could you make sure the work is tracked in #1137?

Reply:
I'd completely missed it because I only looked at the Tasks list in the description. I added another item to that list that points to your comment so now I'm satisfied (feel free to edit further if you wish).